src/librehatti/catalog/urls.py

```diff
@@ -1,37 +1,38 @@
 """
 urls of catalog are..
 """
-from django.conf.urls import url, patterns
-
+#from django.conf.urls import url, patterns
+from django.conf.urls import url
+from . import views as catalog_views
 from django.views.generic import TemplateView
-
+from . import request_change
 
 """
 urls showing the list of item purchased and redirects to page of adding 
 new categories of product 
 """
-urlpatterns = patterns('librehatti.catalog.views',
-    url(r'^$', 'index'),
-    url(r'^select_sub_category/', 'select_sub_category'),
-    url(r'^jsreverse/', 'jsreverse'),
-    url(r'^select_item/', 'select_item'), 
-    url(r'^bill_cal/','bill_cal'),
-    url(r'^list_products/','list_products'),
-    url(r'^previous_value/','previous_value'),
-    url(r'^order_added_success/','order_added_success'),
-    url(r'^change_request/','change_request'),
-    url(r'^price_per_unit/','price_per_unit'),
-    url(r'^select_type/','select_type'),
-    url(r'^nonpaymentorderofsession/','nonpaymentorderofsession'),
-    url(r'^nonpaymentordersuccess/','nonpaymentordersuccess'),
-)
+urlpatterns = [
+    url(r'^$', catalog_views.index, name ='index' ),
+    url(r'^select_sub_category/', catalog_views.select_sub_category, name ='select_sub_category' ),
```

Comment:
missing intendation and extra space found.

Reply:
Would you tell me which space and indentation you are talking about? Please tell the line number.

Reply:
In some lines there is space before ) and in some cases its missing smae goes with comma's

Reply:
@amarjeetkapoor1 Thanks for pointing out. It's been corrected.